update-translations.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+for lang in es fa fr it pl pt_BR tr; {
+    python -m django makemessages -l $lang;
```

Comment:
I tried adding all languages to test settings to make `makemessages --all` work, but got an error about the code `pt_BR`

Reply:
Really strange ...